infrastructure/src/main/java/com/masegosadev/infrastructure/persistence/videogame/VideogameRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.masegosadev.infrastructure.persistence.videogame;
+
+import com.masegosadev.infrastructure.persistence.videogame.model.VideogameEntity;
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.*;
+
+@Repository
+public interface VideogameRepository extends MongoRepository<VideogameEntity,String> {
```

Comment:
We need to use a name for this kind of classes more descriptive. Ex: MongoVideogameRepository

Reply:
Done